app/src/ui/organism/about/About.js

```diff
@@ -0,0 +1,53 @@
+import AboutImg from "../../../assets/img/about-v02.png";
+import Image from "../../atoms/image/Image";
+
+const About = () => {
+  return (
+    <div className="bg-black py-20">
+      <div className="grid grid-rows-1  grid-cols-2 gap-4 container">
+        <Image className="w-full" src={AboutImg} />
+        <div className="ms-24">
+          <div className="flex items-center gap-3 text-primary border border-white rounded-full px-6 py-3 inline-flex mb-8">
+            <span className="uppercase">About Company</span>
+            <span className="icon-arrow-up-left2"></span>
+          </div>
+          <h2 className="text-white text-6xl font-bold leading-[125%] mb-8">
+            We Create Impactful <br /> Digital Experiences
+          </h2>
+          <div className="flex items-center justify-between mb-8">
+            <div className="text-white">
+              <h2 className="text-4xl font-bold">25+</h2>
+              <span className="text-gray-300">Completed projects</span>
+            </div>
+            <p className="text-white">
```

Comment:
text-white daj do rodzica, dzięki temu możesz usunąć className="text-white" - zostanie samo p
To samo dla div-a (rodzeństwa) wyżej

Reply:
wiem